doc/source/getting_started/docker/linux_container.rst

```diff
@@ -0,0 +1,178 @@
+.. _ref_linux_docker:
+
+Geometry Service as a Linux Docker container
+==============================================
+
+.. contents::
+
+.. _ref_running_linux_containers:
+
+Running Linux Docker containers
+---------------------------------
+
+To run the Linux Docker container for the Geometry service, ensure that you follow
+these steps when installing Docker:
+
+.. tab-set::
+
+    .. tab-item:: Linux machines
+
+       If you are on a Linux machine, install `Docker for your distribution <https://docs.docker.com/engine/install/#server>`_
+
+    .. tab-item:: Windows/MacOS machines
+
+       #. Install `Docker Desktop for Windows <https://docs.docker.com/desktop/install/windows-install/>`_ or
+          `Docker Desktop for MacOS <https://docs.docker.com/desktop/install/mac-install/>`_.
+
+       #. **(On Windows)** When prompted for **Use WSL2 instead of Hyper-V (recommended)**, **UNTICK** this checkbox.
```

Comment:
```suggestion
       #. **(On Windows)** When prompted for **Use WSL2 instead of Hyper-V (recommended)**, clear this checkbox.
```
All style guides say you select a check box or clear a check box.